index.html

```diff
@@ -227,6 +227,26 @@ <h3>
             </div>
 
         </section>
+        <section class="contact-page">
+            <div class="contact_items_container">
+                <div class="contact-title">
+                    <h2 class="h2-white">Contact me</h2>
+                    <p class="p-white">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+                </div>
+                <form action="https://formspree.io/f/mzbwnvzb" method="post">
+                    <input type="text" name="Name" id="name" placeholder="Full Name" maxlength="30" required>
```

Comment:
kindly consider using a two-space indentation. This is an agreed upon [html best practice](https://www.freecodecamp.org/news/html-best-practices/).  👍🏾